lib/galaxy/config.py

```diff
@@ -463,6 +463,12 @@ def __init__( self, **kwargs ):
         self.dynamic_proxy_external_proxy = string_as_bool( kwargs.get( "dynamic_proxy_external_proxy", "False" ) )
         self.dynamic_proxy_prefix = kwargs.get( "dynamic_proxy_prefix", "gie_proxy" )
 
+        self.dynamic_proxy = kwargs.get( "dynamic_proxy", "nodejs" )
+        self.dynamic_proxy_golang_noaccess = kwargs.get( "dynamic_proxy_golang_noaccess", 60 )
+        self.dynamic_proxy_golang_clean_interval = kwargs.get( "dynamic_proxy_golang_clean_interval", 10 )
+        self.dynamic_proxy_golang_docker_address = kwargs.get( "dynamic_proxy_golang_docker_address", "unix:///var/run/docker.sock" )
```

Comment:
Hmm... I don't like that there is a different way to kill configure the docker target in a different language. Also the framework already supports different docker hosts for different IEs right?

This is a tricky and substantial request - but I'd like to see communication back to Galaxy on timeout and have a Galaxy responsible for killing the IE. It adds some complexity though - so it might be less robust. What are your thoughts?